support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -255,6 +256,21 @@ export function Checkout({ geoId, appConfig, abParticipations }: Props) {
 		);
 	}, []);
 
+	const checkoutSessionIdUrlParam = 'checkoutSessionId';
+	const maybeCheckoutSessionId = urlSearchParams.get(checkoutSessionIdUrlParam);
+
+	let persistedFormData;
+	if (maybeCheckoutSessionId) {
+		persistedFormData = getFormDetails(maybeCheckoutSessionId);
+
+		// If there's no persisted data, remove the checkoutSessionId from the URL
+		if (!persistedFormData) {
+			const url = new URL(window.location.href);
+			url.searchParams.delete(checkoutSessionIdUrlParam);
+			window.location.href = url.toString();
+		}
+	}
```

Comment:
I would extract this to a function which returns `PersistedFormData | undefined`